pkg/client/cli/output/output.go

```diff
@@ -74,6 +74,18 @@ func Object(ctx context.Context, obj any, override bool) {
 	}
 }
 
+// StreamObject prints an object immediately.
+func StreamObject(ctx context.Context, obj any, override bool) {
```

Comment:
The idea with the `Object` function is that it panics if mixed with any other types of output. The whole point of designing it that way, was that we never should create commands that produces multiple formatted objects, because it leads to unpredictable JSON output.